packages/webapp/src/state/slices/mempool.ts

```diff
@@ -0,0 +1,70 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+interface SettleTx {
+  from: string
+  hash: string
+}
+
+interface BidTx {
+  from: string
+  hash: string
+  value: string
+}
+
+interface MempoolState {
+  listening: boolean
+  pendingSettleTxs: SettleTx[]
+  pendingBidTxs: BidTx[]
+}
+
+const initialState: MempoolState = {
+  listening: false,
+  pendingSettleTxs: [],
+  pendingBidTxs: [],
+};
+
+export const mempoolSlice = createSlice({
+  name: 'mempool',
+  initialState,
+  reducers: {
+    setMempoolListening: (state, action: PayloadAction<boolean>) => {
```

Comment:
See comment in `alchemyMiddleware` about this bit of state

Reply:
Thanks! Removed this one!